routers/web/web.go

```diff
@@ -259,7 +259,13 @@ func RegisterRoutes(m *web.Route) {
 	m.Get("/milestones", reqSignIn, reqMilestonesDashboardPageEnabled, user.Milestones)
 
 	// ***** START: User *****
+	m.Get("/user/kitspace/session", user.GetKitspaceSession)
 	m.Group("/user", func() {
+		m.Group("/kitspace", func() {
+			m.Post("/sign_up", bindIgnErr(forms.RegisterForm{}), user.KitspaceSignUp)
+			m.Post("/sign_in", bindIgnErr(forms.SignInForm{}), user.KitspaceSignIn)
+		})
```

Comment:
```suggestion
	m.Group("/user", func() {
		m.Group("/kitspace", func() {
			m.Get("/session", user.GetKitspaceSession)
			m.Post("/sign_up", bindIgnErr(forms.RegisterForm{}), user.KitspaceSignUp)
			m.Post("/sign_in", bindIgnErr(forms.SignInForm{}), user.KitspaceSignIn)
		})
```

Reply:
I believe this is outside the group because the group is csrf protected. I need to confirm and test this again but that's not really what this PR is about.